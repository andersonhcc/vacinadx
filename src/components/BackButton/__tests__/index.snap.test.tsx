/* eslint-disable react/react-in-jsx-scope */
import {BackButton} from '..';
import renderer from 'react-test-renderer';
import {ThemeProvider} from 'styled-components/native';

import {themeLight as theme} from '../../../constants/styles/themes/light';

describe('BackButton', () => {
  it('should render with icon', () => {
    const {toJSON} = renderer.create(
      <ThemeProvider theme={theme}>
        <BackButton icon="apple" />
      </ThemeProvider>,
    );

    expect(toJSON()).toMatchSnapshot();
  });
});
