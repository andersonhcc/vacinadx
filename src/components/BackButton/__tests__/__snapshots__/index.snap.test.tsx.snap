// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BackButton should render with icon 1`] = `
<View
  accessibilityState={
    {
      "busy": undefined,
      "checked": undefined,
      "disabled": undefined,
      "expanded": undefined,
      "selected": undefined,
    }
  }
  accessibilityValue={
    {
      "max": undefined,
      "min": undefined,
      "now": undefined,
      "text": undefined,
    }
  }
  accessible={true}
  collapsable={false}
  focusable={true}
  onBlur={[Function]}
  onClick={[Function]}
  onFocus={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    [
      {
        "paddingBottom": 10,
        "paddingLeft": 10,
        "paddingRight": 10,
        "paddingTop": 10,
      },
    ]
  }
>
  <Image
    resizeMode="contain"
    size={15}
    source={
      {
        "testUri": "../../../src/assets/icons/apple/Vector.png",
      }
    }
    style={
      [
        {
          "maxHeight": 15,
          "maxWidth": 15,
          "minHeight": 15,
          "minWidth": 15,
        },
      ]
    }
  />
</View>
`;
